server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/sql/TestSqlHostVariable.java

```diff
@@ -163,6 +162,48 @@ public class TestSqlHostVariable {
                   + "          49 {$*VAR|1} USAGE IS SQL TYPE IS XML AS CHARACTER LARGE OBJECT (10).\n"
                   + "        PROCEDURE DIVISION.\n"
                   + "           DISPLAY {$var-name1}.";
+
+  public static final String LOD_VARS_ARRAYS_TEXT1 =
+          "        Identification Division.\n"
+                  + "        Program-Id. 'TEST1'.\n"
+                  + "        Data Division.\n"
+                  + "         Working-Storage Section.\n"
+                  + "       01 {$*VAR1}.\n"
+                  + "          04 {$*VAR5} USAGE IS SQL TYPE IS CLOB (20) OCCURS 30 TIMES.\n"
+                  + "          04 {$*VAR6} USAGE IS SQL TYPE IS DBCLOB (30 K) OCCURS 40.\n"
+                  + "        PROCEDURE DIVISION.\n"
+                  + "           DISPLAY {$VAR1}.\n";
+
+  public static final String LOD_VARS_ARRAYS_TEXT2 =
+          "        Identification Division.\n"
+                  + "        Program-Id. 'TEST1'.\n"
+                  + "        Data Division.\n"
+                  + "         Working-Storage Section.\n"
+                  + "       01 {$*GREET}.\n"
+                  + "          40 {$*VAR} USAGE IS SQL TYPE IS CLOB (10 M) OCCURS 100000 {TIMES|1}.\n"
+                  + "        PROCEDURE DIVISION.\n"
+                  + "           DISPLAY {$VAR}.";
+
+  public static final String LOD_VARS_ARRAYS_TEXT3 =
+          "        Identification Division.\n"
+                  + "        Program-Id. 'TEST1'.\n"
+                  + "        Data Division.\n"
+                  + "         Working-Storage Section.\n"
+                  + "       01 {$*GREET}.\n"
+                  + "          49 {$*VAR|1} USAGE IS SQL TYPE IS CLOB (10 M) OCCURS 35 TIMES.\n"
+                  + "        PROCEDURE DIVISION.\n"
+                  + "           DISPLAY {$VAR}.";
+
+  public static final String LOD_VARS_ARRAYS_TEXT4 =
+          "        Identification Division.\n"
+                  + "        Program-Id. 'TEST1'.\n"
+                  + "        Data Division.\n"
+                  + "         Working-Storage Section.\n"
+                  + "       01 {$*GREET}.\n"
+                  + "          02 {$*VAR1`->VAR1`->VAR1-LENGTH`->VAR1-DATA} USAGE IS SQL TYPE IS BINARY LARGE OBJECT (10) OCCURS 10.\n"
+                  + "        PROCEDURE DIVISION.\n"
+                  + "           DISPLAY {$VAR1-LENGTH}.\n"
+                  + "           DISPLAY {$VAR1-DATA}.\n";
```

Comment:
I am unable to build this one.
- The source overflows column 72.
- After adjusting, I get
```
==000010==> IGYPS2028-E "VAR1-LENGTH" was a table element but was not subscripted or indexed.
                        The first occurrence of the table element was assumed.               
```